### DISCUSSION TITLE
App page shows up blank! ERROR in console: Blocked script execution in '<URL>' because the document's frame is sandboxed and the 'allow-scripts' permission is not set.

### DISCUSSION THREAD

**Discussion by hasnain37** [Kyon147/laravel-shopify#345] [Q&A]
We got warning from shopify to upgrade the App brigde version. previously we are using osiset package in our app.
according to Shopify here are the instructions
https://shopify.dev/docs/api/app-bridge/previous-versions/updating-overview

Now I have update to Kyon laravel shopify package and update the App bridge as well, Now App page is not showing Show this 
![image](https://github.com/user-attachments/assets/9fcda4bf-b9f8-47be-a0e8-43809eece5d7)

Please guide me on this, Thanks in advance
@Kyon147

**Comment by Kyon147** [comment]
You just need to edit your config file - that is the easiest approach but if you are using the older package you might need to migrate first.

As this is not a bug going to convert it to a discussion

**Reply by hasnain37** [reply]
I had migrated already, what next step i do?

**Reply by iamaks1993** [reply]
@Kyon147 @hasnain37 Any solution? I am using V23.0. Thanks

**Reply by Kyon147** [reply]
Have you upgraded from a previous version @iamaks1993? If so, have you followed the upgrade guide?

**Reply by iamaks1993** [reply]
@Kyon147 Thanks for the quick response. It's fresh install. Despite this error in console, the app is working fine.  I found one thread on [shopify community](https://community.shopify.dev/t/blocked-script-execution-in-url-because-the-documents-frame-is-sandboxed-and-the-allow-scripts-permission-is-not-set/11513/4), as per them they are aware of this issue.

**Reply by FaintShadow** [reply]
I am getting that exact same problem and I am new to Shopify so I don't know what I should be looking for in my page. I have the following page which is just for testing if things are working or not:
```.blade.php
@extends('shopify-app::layouts.default')

@section('content')
    <div class="Polaris-Page">
        <div class="Polaris-Page__Content">
            <div class="Polaris-Card">
                <div class="Polaris-Card__Section">
                    <p>You are: <strong>{{ $shopDomain ?? Auth::user()->name }}</strong></p>
                    <button class="Polaris-Button Polaris-Button--pressable Polaris-Button--variantPrimary Polaris-Button--sizeMedium Polaris-Button--textAlignCenter" type="button">
                        <span class="Polaris-Text--root Polaris-Text--bodySm Polaris-Text--medium">Save theme</span>
                    </button>
                </div>
            </div>
        </div>
    </div>
@endsection

@section('scripts')
    @parent
    <script>
        var AppBridge = window['app-bridge'];
        var createApp = AppBridge.default;
        var app = createApp({
            apiKey: '{{ config('shopify-app.api_key') }}',
            shopOrigin: '{{ Auth::user()->name }}',
            forceRedirect: true
        });

        // Then access actions from the app
        var actions = AppBridge.actions;
        var TitleBar = actions.TitleBar;

        var titleBarOptions = {
            title: 'Welcome to My Shopify App',
        };

        var titleBar = TitleBar.create(app, titleBarOptions);
    </script>
@endsection
```
and the error: `Blocked script execution in '[URL]' because the document's frame is sandboxed and the 'allow-scripts' permission is not set.`

**Reply by Kyon147** [reply]
From what I've seen this error doesn't stop an app from loading.

I get this same error in the console but the app loads correctly.

My guess is something else is the actual issue and this error is a red herring.
